docs/hadoop-catalog.md

```diff
@@ -24,9 +24,16 @@ Hadoop 3. If there's any compatibility issue, please create an [issue](https://g
 
 ### Catalog properties
 
-| Property Name | Description                                     | Default Value | Required | Since Version |
-|---------------|-------------------------------------------------|---------------|----------|---------------|
-| `location`    | The storage location managed by Hadoop catalog. | (none)        | No       | 0.5.0         |
+| Property Name                                      | Description                                                                                                                                                         | Default Value | Required                     | Since Version |
+|----------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|------------------------------|---------------|
+| `location`                                         | The storage location managed by Hadoop catalog.                                                                                                                     | (none)        | No                           | 0.5.0         |
+| `authentication.enable`                            | Whether to enable authentication for Hadoop catalog.                                                                                                                | false         | No                           | 0.5.1         |
+| `authentication.type`                              | The type of authentication for Hadoop catalog, currently we only support `kerberos`. `authentication.enable` should be true to make this configuration take effect. | (none)        | No                           | 0.5.1         |
+| `authentication.kerberos.impersonation-enable`     | Whether to enable impersonation for the Hadoop catalog.                                                                                                             | false         | No                           | 0.5.1         |
```

Comment:
impersonation-enable should be unrelated to authentication.
We use simple authentication mode. we should also use impersonation-enable.